mongodbatlas/cloud_provider_snapshots.go

```diff
@@ -43,18 +43,22 @@ var _ CloudProviderSnapshotsService = &CloudProviderSnapshotsServiceOp{}
-	Description      string  `json:"description,omitempty"`      // Description of the on-demand snapshot.
-	Links            []*Link `json:"links,omitempty"`            // One or more links to sub-resources and/or related resources.
-	MasterKeyUUID    string  `json:"masterKeyUUID,omitempty"`    // Unique ID of the AWS KMS Customer Master Key used to encrypt the snapshot. Only visible for clusters using Encryption at Rest via Customer KMS.
-	MongodVersion    string  `json:"mongodVersion,omitempty"`    // Version of the MongoDB server.
-	SnapshotType     string  `json:"snapshotType,omitempty"`     // Specified the type of snapshot. Valid values are onDemand and scheduled.
-	Status           string  `json:"status,omitempty"`           // Current status of the snapshot. One of the following values: queued, inProgress, completed, failed
-	StorageSizeBytes int     `json:"storageSizeBytes,omitempty"` // Specifies the size of the snapshot in bytes.
-	Type             string  `json:"type,omitempty"`             // Specifies the type of cluster: replicaSet or shardedCluster.
+	ID               string    `json:"id,omitempty"`               // Unique identifier of the snapshot.
+	RetentionInDays  int       `json:"retentionInDays,omitempty"`  // The number of days that Atlas should retain the on-demand snapshot. Must be at least 1 .
+	CreatedAt        string    `json:"createdAt,omitempty"`        // UTC ISO 8601 formatted point in time when Atlas took the snapshot.
+	ExpiresAt        string    `json:"expiresAt,omitempty"`        // UTC ISO 8601 formatted point in time when Atlas will delete the snapshot.
+	Description      string    `json:"description,omitempty"`      // Description of the on-demand snapshot.
+	Links            []*Link   `json:"links,omitempty"`            // One or more links to sub-resources and/or related resources.
+	MasterKeyUUID    string    `json:"masterKeyUUID,omitempty"`    // Unique ID of the AWS KMS Customer Master Key used to encrypt the snapshot. Only visible for clusters using Encryption at Rest via Customer KMS.
+	MongodVersion    string    `json:"mongodVersion,omitempty"`    // Version of the MongoDB server.
+	SnapshotType     string    `json:"snapshotType,omitempty"`     // Specified the type of snapshot. Valid values are onDemand and scheduled.
+	Status           string    `json:"status,omitempty"`           // Current status of the snapshot. One of the following values: queued, inProgress, completed, failed
+	StorageSizeBytes int       `json:"storageSizeBytes,omitempty"` // Specifies the size of the snapshot in bytes.
+	Type             string    `json:"type,omitempty"`             // Specifies the type of cluster: replicaSet or shardedCluster.
+	CloudProvider    string    `json:"cloudProvider,omitempty"`    // Cloud provider that stores this snapshot. Atlas returns this parameter when "type": "replicaSet.
```

Comment:
[nit, typo], `"type": "replicaSet` -> `"type": "replicaSet"`

Reply:
Update, thank you